Postfix_Evaluation/Postfix _Evaluation.c

```diff
@@ -0,0 +1,87 @@
+//Postfix Evaluation
+#include<stdio.h>
+#include<string.h>
+#include<ctype.h>
+
+struct stack
+{
+   char A[100];
+   int Top;
+};
+
+void push(struct stack *ps, char x)
+{
+   if(ps -> Top == 99)
+   printf("\nStack is Full\n");
+   else
+   {
+   ps -> Top = ps -> Top + 1;
+   ps -> A[ps -> Top] = x;
+   }
+}
+
+int evaluate(int op1, int op2, char c)
+{
+   if(c == '+')
+   return op1 + op2;
+   else if(c == '-')
+   return op1 - op2;
+   else if(c == '*')
+   return op1 * op2;
+   else if(c == '/')
+   return op1 / op2;
+}
+
+void print(struct stack s)
+{
+   int i;
+   for(i = 0 ; i <= s.Top ; i++)
+   printf("%d\t", s.A[i]);
+}
+
+int pop(struct stack *ps)
+{
+   int x;
+   if(ps->Top == -1)
+   printf("Stack is empty!Element cant be deleted\n");
+   else
+   {
+     x = ps -> A[ps -> Top];
+     ps -> Top = ps -> Top - 1;
+     return x;
+   }
+}
+
+int main()
+{
+  int op1, op2, x, i;
```

Comment:
Please correct the indentation(4 space indentation)

Reply:
@somya-kapoor Corrected the indentation to 4 spaces